### ISSUE TITLE
why we remove script or style tag?

### ISSUE THREAD

**Issue by calebxx** [Pajamaman/clean-html#12]
which is really import to me...
```
import $ from 'jquery'
import { clean } from 'clean-html'

const $code = $('code.api-preview')
const badHtml = $code.text()
const opts = {
    'add-break-around-tags': ['figure', 'iframe', 'a', 'img', 'time', 'address', 'details', 'script', 'ul', 'ol', 'li'],
    'wrap': 0
}

clean(badHtml, opts, (clearHtml) => {
    $('code.api-preview').text(clearHtml)
        .css('font-size', '14px')
        .css('font-family', 'Hack, "DejaVu Sans Mono", Menlo, Consolas, "Liberation Mono", Monaco, "Lucida Console", monospace')
        .css('line-height', '20px')
})
```
now i need to replace style and style tag into ```ruby``` tag, and replace it back after cleaning.
INTO this:
```
const $code = $('code')
const badHtml = $code.text()
    .replace('<script', '<ruby')
    .replace('\/script>', '\/ruby>')
    .replace('<style', '<dialog')
    .replace('\/style>', '\/dialog>')

const opts = {
    'add-break-around-tags': [
        'figure', 'iframe', 'a', 'img', 'time', 'address', 'details', 'script', 'ul', 'ol', 'li', 'ruby', 'dialog'
    ],
    'wrap': 0
}

clean(badHtml, opts, (clearHtml) => {
    let newClearHtml = clearHtml.replace('<ruby', '<script')
        .replace('\/ruby>', '\/script>')
        .replace('<dialog', '<style')
        .replace('\/dialog>', '\/style>')

    $('code.api-preview').text(newClearHtml)
        .css('font-size', '14px')
        .css('font-family', 'Hack, "DejaVu Sans Mono", Menlo, Consolas, "Liberation Mono", Monaco, "Lucida Console", monospace')
        .css('line-height', '20px')
})
```
would we made it remove-able by configure?

**Comment by dave-kennedy** [comment]
From the release notes for version 1.4.1:

> Trying to preserve CSS and JavaScript formatting is a pain, so style and script tags are no longer supported in this release. They will simply be removed from the output.

I'd prefer to leave them alone, as I'm not interested in trying to clean anything in a script or a style tag. But, because of the way the script works, that's actually really difficult.

I may take another look at it. Of course, a PR from you wouldn't hurt if you're willing to try.

**Comment by calebxx** [comment]
yes. you are right. things inside of those tags is hard to deal.

**Comment by dave-kennedy** [comment]
Fixed in 6a1cc0d.
